platform/platform-app/src/main/resources/features.xml

```diff
@@ -1046,15 +1046,25 @@
         <feature>platform-migratable</feature>
     </feature>
 
+    <feature name="netty-core" install="manual" version="${netty.version}"
+             description="Netty libraries">
+        <bundle>mvn:io.netty/netty-common/${netty.version}</bundle>
+        <bundle>mvn:io.netty/netty-resolver/${netty.version}</bundle>
+        <bundle>mvn:io.netty/netty-transport/${netty.version}</bundle>
+        <bundle>mvn:io.netty/netty-buffer/${netty.version}</bundle>
+        <bundle>mvn:io.netty/netty-codec/${netty.version}</bundle>
+        <bundle>mvn:io.netty/netty-handler/${netty.version}</bundle>
+    </feature>
+
     <feature name="artemis-core" install="manual" version="${artemis.version}" description="ActiveMQ Artemis broker libraries">
-        <feature prerequisite="true">transaction</feature>
-        <feature prerequisite="true">netty-core</feature>
-        <feature prerequisite="true">scr</feature>
+        <feature>transaction</feature>
+        <feature>netty-core</feature>
+        <feature>scr</feature>
         <configfile finalname="etc/org.apache.activemq.artemis.cfg">mvn:org.apache.activemq/artemis-features/${artemis.version}/cfg</configfile>
         <configfile finalname="etc/artemis.xml">mvn:org.apache.activemq/artemis-features/${artemis.version}/xml/artemis</configfile>
 
         <bundle>mvn:org.apache.geronimo.specs/geronimo-jms_2.0_spec/${geronimo-jms_2_spec.version}</bundle>
-        <bundle dependency="true">mvn:com.google.guava/guava/${guava.version}</bundle>
+        <bundle>mvn:com.google.guava/guava/19.0</bundle>
```

Comment:
yes and things didn't work with the newer version if I tried to upgrade ddf's guava, but not in obvious fixable ways or I would have just fixed them.